docker/tests/Templates/centos7/Dockerfile

```diff
@@ -16,7 +17,7 @@ ENV LANG en_US.UTF-8
 ENV LC_ALL $LANG
 RUN localedef --charmap=UTF-8 --inputfile=en_US $LANG
 
-RUN curl -L -o powershell-$PSVERSIONSTUB-1.rhel.7.x86_64.rpm $PACKAGELOCATIONSTUB/powershell-$PSVERSIONSTUB-1.rhel.7.x86_64.rpm
-RUN yum install -y powershell-$PSVERSIONSTUB-1.rhel.7.x86_64.rpm
-RUN git clone --recursive $GITLOCATION
-RUN pwsh -c "Import-Module /PowerShell/build.psm1;\$dir='/usr/local/share/powershell/Modules';New-Item -Type Directory -Path \$dir -ErrorAction SilentlyContinue;Restore-PSPester -Destination \$dir;exit (Invoke-Pester $TESTLISTSTUB -PassThru).FailedCount"
+RUN curl -L -o $PACKAGENAME $PACKAGELOCATION/$PACKAGENAME
+RUN yum install -y $PACKAGENAME
+RUN $TESTDOWNLOADCOMMAND
+RUN pwsh$PREVIEWSUFFIX -c "Import-Module /PowerShell/build.psm1;\$dir='/usr/local/share/powershell/Modules';New-Item -Type Directory -Path \$dir -ErrorAction SilentlyContinue;Restore-PSPester -Destination \$dir;exit (Invoke-Pester $TESTLIST -PassThru).FailedCount"
```

Comment:
Maybe `Start-PSPester` might be better instead of `Invoke-Pester` as it loads helpers. It uses `-Path` parameter for the test list.

Reply:
After we discussed this, I could not make it work. We'll need to dig deeper later; for this PR this change is not required.